apps/nova/src/components/TutorSettings/ProfileForm.tsx

```diff
@@ -0,0 +1,24 @@
+import { TutorProfileCard } from "@litespace/luna/TutorProfile";
+import React from "react";
+import { useFindTutorInfo } from "@litespace/headless/tutor";
+import { Button, ButtonSize } from "@litespace/luna/Button";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+
+import { TutorSettingsTabs } from "@/components/TutorSettings/SettingsTabs";
+
+export const ProfileForm: React.FC<{
```

Comment:
The term `form` is misleading. I believe we can come up with a better descriptive name.